apps/pie-docs/src/components/text-input/overview/overview.md

```diff
@@ -0,0 +1,492 @@
+---
+eleventyNavigation:
+    key: Overview
+    parent: 'Text Input'
+    order: 1
+shouldShowContents: true
+---
+
+## Overview
+The purpose of text inputs is to gather user input in the form of text, enabling them to provide specific information, such as names, addresses, messages or any other textual data required by the application or website.
+
+Text inputs are fundamental elements used in forms, login screens and any user interface where users need to provide textual information.
+
+{% contentPageImage {
+    src:"../../../assets/img/components/text-input/overview.svg",
+    alt: "Two text inputs that are vertically stacked together."
+} %}
+
+---
+
+## Dos and Don’ts
+
+{% usage {
+    do: {
+        type: usageTypes.text,
+        items: [
+          "Use it when you want the user to enter a single-line value."
+        ]
+    },
+    dont: {
+        type: usageTypes.text,
+        items: [
+            "Don’t use if it is likely that the input will exceed one line, if this is the case use a [textarea](/components/textarea/) instead."
+        ]
+    }
+} %}
+
+---
+
+## Anatomy
+
+{% contentPageImage {
+    src:"../../../assets/img/components/text-input/anatomy.svg",
+    alt: "Anatomy of a text input.",
+    width: 420
+} %}
+
+{% list {
+    type: listTypes.ordered,
+    items: [
+        "**Form label (optional):** The [form label](/components/form-label/) provides necessary content / information to a form.",
+        "**String:** Placeholder or user inputted value.",
+        "**Leading content (optional):** An icon, text, or symbol that is used to visually support the input. It is displayed before the input’s value. ",
+        "**Trailing content (optional):** An icon, text, or symbol that is used to visually support the input. It is displayed after the input's value.",
+        "**Assistive text (optional):** The [assistive text](components/assistive-text/) provides additional instructional information / error / success messaging."
```

Comment:
Currently the link goes to https://pr1500-docs.pie.design/components/text-input/overview/components/assistive-text/. The form label link works fine though so adding the leading `/` should fix it.
```suggestion
        "**Assistive text (optional):** The [assistive text](/components/assistive-text/) provides additional instructional information / error / success messaging."
```